### PR TITLE
fix(scylla_bench_thread): prop up s-b version

### PR DESCRIPTION
**Pull request by dkropachev**
https://trello.com/c/eE4dRKSE/3724-prop-up-scylla-bench-version-on-sct

s-b got a new tag - https://github.com/scylladb/scylla-bench/releases/tag/v0.1.3
Let's prop version number up on sct, also we need to add additional options that were introduced by following commits:
https://github.com/scylladb/scylla-bench/commit/5662afcdd7ac81a5660679e87de3accf3ccf018e
https://github.com/scylladb/scylla-bench/commit/d1644082c188494a031eddb710ecf588b3913f44

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] ~~New configuration option are added and documented (in `sdcm/sct_config.py`)~~
- [ ] ~~I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)~~
- [x] All new and existing unit tests passed (CI)
- [ ] ~~I have updated the Readme/doc folder accordingly (if needed)~~

### CODE REVIEW

**Comment by dkropachev** [comment]
Test run: https://jenkins.scylladb.com/view/staging/job/scylla-staging/job/dmitry/job/longevity-twcs-3h/3/execution/node/117

File: sdcm/cluster.py [hunk 1]
```diff
@@ -4729,7 +4729,7 @@ def install_scylla_bench(node):
             echo 'export GOPATH=$HOME/go' >> $HOME/.bash_profile
             echo 'export PATH=$PATH:/usr/local/go/bin' >> $HOME/.bash_profile
             source $HOME/.bash_profile
-            GO111MODULE=on go get -v github.com/scylladb/scylla-bench@v0.1.2
+            GO111MODULE=on go get -v github.com/scylladb/scylla-bench@v0.1.3
```
Comment (juliayakovlev) [inline sdcm/cluster.py:4732]: Maybe we need to put s-b github repository in `defaults/test_default.yaml`? Not continue hold it in the code

File: sdcm/nemesis.py [hunk 2]
```diff
@@ -161,7 +161,6 @@ def __init__(self, tester_obj, termination_event, *args):  # pylint: disable=unu
         self._add_drop_column_columns_info = {}
         self._add_drop_column_target_table = []
         self._add_drop_column_tables_to_ignore = {
-            'scylla_bench': '*',  # Ignore scylla-bench tables
```
Comment (juliayakovlev) [inline sdcm/nemesis.py:164]: Why do you remove it?

Comment (dkropachev) [reply sdcm/nemesis.py:164]: It is all in commit message, new version of s-b supports column adding/removing

File: sdcm/cluster.py (hunk 1 above)
Comment (dkropachev) [reply sdcm/cluster.py]: added

**Review by juliayakovlev — APPROVED** [review]

**Review by bentsi — APPROVED** [review]

**Pull request merged**